esp/esp/utils/widgets.py

```diff
@@ -219,6 +219,8 @@ def render(self, name, value, attrs=None, choices=()):
         return mark_safe(output)
 
 class NavStructureWidget(forms.Widget):
+    # TODO(benkraft): Convert this to an actual static script so we don't have
```

Comment:
I have no clue what's going on in this file. Have you checked that it doesn't break the other themes' forms? If so, I think I'm willing to not worry about it too much, since it was already like this.